cxx-sensors/src/main/java/org/sonar/cxx/sensors/compiler/vc/CxxCompilerVcParser.java

```diff
@@ -96,7 +67,7 @@ public void processReport(final SensorContext context, File report, String chars
 
   private static String removeMPPrefix(String fpath) {
```

Comment:
Do we really need this MP prefix removal as an extra step? Could be part of global regex?